### ISSUE TITLE
Numeric validation for price tags in marketplace add

### ISSUE THREAD

**Issue by kibcode** [PHPfox-Official/phpfox-v4-issues#2723]
## What's happened?
when using e.g. euro as currency in marketplace we now can format the price tag in currency settings. it works to show prices in euro like this 42,00 on item view and brose sections but it does not work when adding or editing a marketplace item. the validation of the price checks for numeric values and does only work with "." and returns false when using ",". also the default value for marketplace items is "0.00" even when currency format is something else.

## Steps to reproduce:
1. change format of currency from {0} #,###.00 {1} to {0} #.###,00 {1}
2. add new marketplace item (you will get default value 0.00)
3. add price e.g. 42,99

## What's expected?
the validation should also allow "," for prices. is_numeric only allows "." to be true. maybe you could add a str_replace for currencies using comma?

default value of marketplace add should change with currency format.

## phpFox version
phpFox 4.7.6
marketplace 4.6.3

**Comment by harrison05** [comment]
Currently, phpFox is using numeric validate for price field, so it can't support "," as decimal separator.

>> the validation should also allow "," for prices. is_numeric only allows "." to be true. maybe you could add a str_replace for currencies using comma?

This might a solution but it not good when editing because price must save price into the database as a decimal value and it doesn't allow "," 

We must discuss this case and find the best solution. Please using "." with the current version.

Regards.

**Comment by kibcode** [comment]
thanks @harrison05!

for workaround i have added a str_replace right before the validation and before asigning the item to template... both in addcontroller. the default value is hardcoded and can be changed too. this works in my case but i know that it is a bit more work to make this dynamic when using more than one currency ;)

**Comment by harrison05** [comment]
Yes, your solution only works if using one currency. So we need to discuss to find a solution for this case.
